source/renderer/app/api/ada/adaTxFee.js

```diff
@@ -13,13 +18,15 @@ export type AdaTxFeeParams = {
 };
 
 export const adaTxFee = (
-  { ca, sender, receiver, amount, groupingPolicy }: AdaTxFeeParams
+  { apiParams, sender, receiver, amount, groupingPolicy }: AdaTxFeeParams
 ): Promise<AdaTransactionFee> => (
   request({
     hostname: 'localhost',
     method: 'POST',
     path: `/api/txs/fee/${sender}/${receiver}/${amount}`,
-    port: 8090,
-    ca,
+    port: apiParams.port,
+    ca: apiParams.ca,
+    cert: apiParams.clientCert,
+    key: apiParams.clientKey,
```

Comment:
@cleverca22, this makes me wonder, if we can refactor this API parametrisation, so we don't have to change the humongous bunch of files every time we change `apiParams`?  Sure we'll have to do go through the global change now, but maybe in such a way that we can avoid this in future?

Maybe @nikolaglumac can suggest something?

Reply:
I mean -- we're going through a very explicit packing and re-packing of `apiParams` everywhere -- maybe we can pass `apiParams` directly?